apps/discord-bot/index.ts

```diff
@@ -31,9 +32,10 @@ const client = new DiscordClient({
 
 const handlersPath = join(__dirname, "handlers");
 const handlerFiles = readdirSync(handlersPath);
-handlerFiles.forEach((handlerFile: any) => {
+handlerFiles.forEach(async (handlerFile: string) => {

```

Comment:
Why did we use async here?

Reply:
I refactored import.then() with await import at the following lines (line 37) and because of that I had to make function async.

Reply:
When we use async here, the event and command files may not be loaded in a sequence (I have no idea, that's just what I see when I look at the console logs), so I think it can give us trouble during a file loading issue and make it difficult to track in the logs.